pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>1.398</version>
+        <version>1.568</version>
```

Comment:
Thanks @armfergom , this version is because of https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/scm/ChangeLogParser.java#L45